src/app/app.module.ts

```diff
@@ -30,6 +30,9 @@ import { DeleteConfirmationDialogComponent } from './shared/delete-confirmation-
 import { SkillUserComponent } from './skill-users/skill-user.component';
 import { SkillGroupsEditComponent } from './skill-groups/skill-groups-edit.component';
 import { SkillGroupsNewComponent } from './skill-groups/skill-groups-new.component';
+import { ProjectsComponent } from './projects/projects.component';
+import { NewProjectComponent } from './projects/new-project/new-project.component';
+import { EditProjectComponent } from './projects/edit-project/edit-project.component';
```

Comment:
I think it would be good to place `edit-project.component` in the same directory with `projects.component` as it is done for `my-skills` components. So it will be one nested level `/projects` instead of `/projects/edit-project`.

Reply:
done